### PR TITLE
Plug search improvements

### PR DESCRIPTION
**Pull request by peteruithoven**
Alterations to get better results when searching for sub settings. See: https://github.com/elementary/applications-menu/pull/133

### CODE REVIEW

File: src/mouse-touchpad.vala [hunk 1]
```diff
@@ -95,7 +95,17 @@ namespace MouseTouchpad {
 
         /* 'search' returns results like ("Keyboard → Behavior → Duration", "keyboard<sep>behavior") */
         public override async Gee.TreeMap<string, string> search (string search) {
-            return new Gee.TreeMap<string, string> (null, null);
+          var search_results = new Gee.TreeMap<string, string> ((GLib.CompareDataFunc<string>)strcmp, (Gee.EqualDataFunc<string>)str_equal);
```
Comment (danrabbit) [inline src/mouse-touchpad.vala]: Looks like this whole section has 2 space indentation instead of 4

File: src/mouse-touchpad.vala [hunk 2]
```diff
@@ -95,7 +95,17 @@ namespace MouseTouchpad {
...
+          search_results.set ("%s → %s".printf (display_name, _("Primary button")), "");
+          search_results.set ("%s → %s".printf (display_name, _("Reveal pointer")), "");
+          search_results.set ("%s → %s".printf (display_name, _("Middle click paste")), "");
+          search_results.set ("%s → %s".printf (display_name, _("Pointer speed")), "");
+          search_results.set ("%s → %s".printf (display_name, _("Tab to click")), "");
```
Comment (danrabbit) [inline src/mouse-touchpad.vala]: This should be "Tap" to click, not "Tab"

File: src/mouse-touchpad.vala (hunk 1 above)
Comment (peteruithoven) [reply src/mouse-touchpad.vala]: Damn, I'll go over my other PR's

**Comment by danrabbit** [comment]
Marking as "Blocked" since it looks like we're waiting for a stable Juno release

**Pull request merged**